test_programs/execution_panic/oracle_return_vector_then_vector/src/main.nr

```diff
@@ -0,0 +1,13 @@
+// Currently the AVM doesn't offer a way to correctly handle the return of multiple vectors from foreign calls,
```

Comment:
Let's just track that we should add a frontend error for this

Reply:
https://github.com/noir-lang/noir/issues/10607

I'll do this now.